scripts/loadable_app/hello.c

```diff
@@ -0,0 +1,12 @@
+int a = 9;
+int b = 9;
+
+int x()
+{
+return 7;
+}
+int main(int argc, char **argv)
+{
+	int p = x();
+	return a + p + x();
```

Comment:
It seems,`i386-elf-gcc hello2.c -o hello -nostdlib -emain -fpie -N` works. I'm not sure it is a good idea to use `-nostdlib`, but maybe it is ok for a stub